cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -166,27 +168,29 @@ def get_ha_appliances_with_providers(unconfigured_appliances, app_creds):
     app1_ip = appl2.hostname
     pwd = app_creds['password']
     # Configure first appliance as dedicated database
-    command_set = ('ap', '', '5', '1', '1', '1', 'y', pwd, TimedCommand(pwd, 360), '')
+    command_set = ('ap', '', '7', '1', '1', '2', 'y', pwd, TimedCommand(pwd, 360), '')
     appl1.appliance_console.run_commands(command_set)
     wait_for(lambda: appl1.db.is_dedicated_active)
     # Configure EVM webui appliance with create region in dedicated database
-    command_set = ('ap', '', '5', '2', app0_ip, '', pwd, '', '2', '0', 'y', app0_ip, '', '', '',
-        pwd, TimedCommand(pwd, 360), '')
+    command_set = ('ap', '', '7', '2', app0_ip, '', pwd, '', '2', '0', 'y', app0_ip, '', '', '',
+        TimedCommand(pwd, 360), '')
     appl3.appliance_console.run_commands(command_set)
     appl3.evmserverd.wait_for_running()
     appl3.wait_for_web_ui()
     # Configure primary replication node
-    command_set = ('ap', '', '6', '1', '1', '', '', pwd, pwd, app0_ip, 'y',
+    command_set = ('ap', '', '8', '1', '1', '', '', pwd, pwd, app0_ip,
```

Comment:
_Optional_ formatting -> moving `        TimedCommand('y', 60), '')` to this line.